cli_tools_e2e_test/common/utils/utils.go

```diff
@@ -270,3 +270,32 @@ func ContainsSubString(strs []string, s string) bool {
 	}
 	return false
 }
+
+// ContainsAll checks whether all given strings in subarr exists in arr
+func ContainsAll(arr []string, subarr []string) bool {
```

Comment:
Can we have these two in common common module somewhere? It's not testing related and would be useful in prod code